trade-finance-manager-api/src/v1/api.js

```diff
@@ -827,6 +827,30 @@ const getPartyDbInfo = async ({ companyRegNo }) => {
   }
 };
 
+/**
+ * Calls getOrCreatePartyDbInfo in external-api to get a customer in Salesforce, and create it if it doesn't exist
+ * @param {number} companyRegNo Party URN
+ * @param {number} companyName Company name
+ * @returns {Promise<Object>} Company information
+ */
+const getOrCreatePartyDbInfo = async ({ companyRegNo, companyName }) => {
+  try {
+    const response = await axios({
+      method: 'post',
+      url: `${EXTERNAL_API_URL}/party-db`,
+      headers: headers.external,
+      data: {
+        companyRegNo,
+        companyName,
+      },
+    });
+    return response.data;
```

Comment:
Should we validate `response` in case it in falsy?

Reply:
I would suggest add optional chaining to be on the safe side please.

Reply:
Cool, no problem have done